book/visual-effects.md

```diff
@@ -1695,73 +1699,64 @@ highly recommend it.
 :::
 
 
-Exercises
-=========
-
-*z-index*: Right now, the order of paint is a depth-first traversal of the
```

Comment:
How about putting it back and just say that it also requires position:relative, which should be implemented,
except that we won't ask them to support top and left?

Reply:
The issue is that even we add `position:relative` we can't create any overlaps between elements with the minimal CSS support we have. `top`/`left`, `transform`, `width`/`height`, and many other options are enough. I'll write it in a more general sense and add it back.

Reply:
I added it back, take a look